generate_adapter.py

```diff
@@ -65,12 +65,14 @@ def main(
     ):
         print("Loading model ...", file=sys.stderr)
         t0 = time.time()
-        model = LLaMA(LLaMAConfig())  # TODO: Support different model sizes
 
         # 1. Load the pretrained weights
         pretrained_checkpoint = torch.load(pretrained_path, map_location=torch.device("cpu"))
+        name = llama_model_lookup(pretrained_checkpoint)
+        model = LLaMA.from_name(name)
```

Comment:
This isn't supported for the adapter version of LLaMA.
Let's do `LLaMAConfig.from_name()` instead here, and pass in the config.

Reply:
I'm not sure I understand why it's not supported though. Aren't we loading the unchanged frozen weights?

Reply:
I'm just referring to the "from_name" function which either needs to be added or be avoided. Note that the LLaMA from model.py and LLaMA from adapter.py is different and thus code can easily get out of sync.

Reply:
I can send a PR to rectify this inconvenience.

Reply:
I'm addressing it here: #170
Hopefully the last thing to unblock this PR :)